tests/networkpolicy_test.go

```diff
@@ -99,13 +99,13 @@ var _ = Describe("[rfe_id:150][crit:high][vendor:cnv-qe@redhat.com][level:compon
 			Expect(err).ToNot(HaveOccurred())
 		})
 
-		It("[test_id:CNV-1511] should be failed to reach vmia from vmib", func() {
+		It("[test_id:1511] should be failed to reach vmia from vmib", func() {
```

Comment:
The test verifies what is in the test-case description, but the test steps says otherwise.
@dshchedr would you please take a look?

For the reference: https://polarion.engineering.redhat.com/polarion/#/project/CNV/workitem?id=CNV-1511

Reply:
@petrkotas better to address this question to network team, @duyanyan can you please take a look? as for me - it seems like mistake in testcase steps

Reply:
Fixed.